File: 01_Language/01_Functions/python/ctype_alpha.py
# coding: utf-8

strings = ['KjgWZC', 'arf12', '']
for s in strings:
    if s.isalpha():
        print("The string " + s + " consists of all letters.")
    else:
        print("The string " + s + " does not consist of all letters.")

File: 01_Language/01_Functions/php/ctype_alpha.php
<?php

$strings = array('KjgWZC', 'arf12', '', null);
foreach ($strings as $testcase) {
    if (ctype_alpha($testcase)) {
        echo "The string $testcase consists of all letters.\n";
    } else {
        echo "The string $testcase does not consist of all letters.\n";
    }
}
